x-pack/plugins/apm/public/components/shared/charts/CustomPlot/getEmptySeries.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { memoize } from 'lodash';
+import d3 from 'd3';
+
+export const getEmptySeries = memoize(
```

Comment:
Are we sure we want to memoize this? If the user has auto-refresh on a 1s interval, this will quickly add up. (Not sure how often that happens, or how costly this code is).

Reply:
That's a good point. So this could build up a lot of cached objects - basically a memory leak.
Since I haven't changed this (just moving the function to another file) and this issue is a blocker which needs to go into 7.2 I'm going to merge as-is but we can remove it in a follow-up PR.